crick/tests/test_stats.py

```diff
@@ -22,8 +22,10 @@
 RTOL = 0
 ATOL = 1e-8
 
+scipy_xfail_mark = pytest.mark.xfail(reason="modern scipy return NaN")
```

Comment:
I'm still a little uncertain here. Did `scipy` change its output and `crick` no longer matches the more recent `scipy` behavior? Or do we just need to update the tests?

Reply:
scipy changed it's behavior, so we'll have to update crick to match the new behavior. Alternatively, we could hardcode the expected values instead of using scipy to generate them, but that's not a very attractive solution to me. The assumption there is that scipy had good reason to change the behavior of these edge cases.